spec/features/admin_deletes_user_spec.rb

```diff
@@ -1,33 +1,26 @@
 require 'rails_helper'
 
-feature 'admin deletes user', %Q{
+feature 'admin deletes article', %Q{
   As an admin
-  I want to delete an user
-  So that I can filter out users who don't want to be on the site
+  I want to delete an article
+  So that I can filter out unwanted articles
 } do
-  
-  scenario 'admin deletes an user' do
-    admin = FactoryGirl.create(:admin_user)
-    article = FactoryGirl.create(:article)
-    user = FactoryGirl.create(:user)
-    
-    sign_in_as(user)
-    add_new_article(article)
-    
-    click_link 'Sign Out'
-    expect(page).to have_content('Signed out successfully')
 
-    sign_in_as(admin)
+  let!(:admin) { FactoryGirl.create(:admin_user) }
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:user1) { FactoryGirl.create(:user) }
 
-    visit edit_admin_article_path(article)
-    click_on 'Delete article'
-    expect(page).to have_content "Article successfully deleted"
-    visit articles_path
-    expect(page).to_not have_content article.name
+  scenario 'admin deletes an user' do
+    sign_in_as(admin)
+    visit edit_admin_user_path(user)
+    click_on 'Delete user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.